pm_icecon/nt/compute_nt_ic.py

```diff
@@ -177,34 +177,93 @@ def apply_nt_spillover(
     *, conc: npt.NDArray, shoremap: npt.NDArray, minic: npt.NDArray
 ) -> npt.NDArray[np.int16]:
     """Apply the NASA Team land spillover routine."""
+    """
+      conc is 0 to > 100.0
+        (per this code, negative value means missing data)
+
+      shoremap is 0: ocean, 1: land, 2: coast,
+                   3: coast, 4: near_coast, 5: far_coast
+        in original, shoremap is big-endian int16, but nothing in the
+          computation requires it to be anything other than int-like
+
+      minic is 0 to 100.0
+        (in original binary file, minic is 0-1000 which is conc * 10)
```

Comment:
Same as above comment - lets be clear what "original" means here, or lets remove this note.